### PR TITLE
Add remove

### PR DESCRIPTION
**Pull request by jsug9**
- When the enter key is pressed on the placeholder, a new to-do is added with the length of the array as the index.
- To-dos can be deleted.
- To-dos can be edited.
- To-dos can be marked or unmarked as completed.
- All changes are saved to localStorage
- When a to-do is deleted, its index is updated
(It appears that linters are not working on pull requests, kindly run them locally)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,16 +1,73 @@
...
+  todos.add({
+    description: 'Brush my teeth',
+    isComplete: true,
+    index: 1,
+  });
+  todos.add({
+    description: 'Finish the assignment',
+    isComplete: false,
+    index: 2,
+  });
```
Comment (ShoiraTa) [inline src/index.js:21]: - [ ] Please kindly remove  hardcoded items from the tasks array.

File: src/index.js [hunk 2]
```diff
@@ -1,16 +1,73 @@
...
+      if (event.keyCode === 13) {
+        todos.add({
+          description: todoInput.value,
+          isComplete: false,
+          index: todos.todos.length,
+        });
+        todos.save();
+        todoInput.value = '';
+        window.location.reload();
+      }
```
Comment (ShoiraTa) [inline src/index.js:70]: - [ ] The property `index` should be set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). Please start giving indexed from 1, currently it starts from 0

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @jsug9 ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (jsug9) [reply src/index.js]: Dear @ShoiraTa  thank you for the very detailed example! I didn't understand that part very well.

Comment (jsug9) [reply src/index.js]: @ShoiraTa no no, your example was very clear, that helped me understand it, thank you!

Comment (ShoiraTa) [reply src/index.js]: oh ok, I misunderstood this time 😄

**Review by od-c0d3r — APPROVED** [review]
Asslam Alaikum @jsug9  ,😊

Hi👋🏽, 
Thanks for implementing the changes required by the previous code reviewer :100: 
Your project is complete!  
There is nothing else to say other than... it's time to merge it 😎 

# Congratulations! 🎉 

Happy coding!👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**